components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -28,20 +28,39 @@ use topology::Topology;
 use manifestjson::ManifestJson;
 use bind;
 
+/// Represents a Kubernetes manifest.
 #[derive(Debug, Clone)]
 pub struct Manifest {
+    /// The service name in the Kubernetes cluster.
     pub metadata_name: String,
+    /// The service name in the Habitat ring.
     pub habitat_name: String,
+    /// The docker image.
```

Comment:
Should there be a separate field for the tag?

Reply:
Not sure I follow. What tag? This is the same as the image name: origin/service

Reply:
The docker tag. It could in theory be an additional `String` field. Or we could just expect it to be present (or not) in this field. Not sure about the pros/cons.

Reply:
ah yes. it should be a String but we'll address that later.